langgraph/src/pregel/read.ts

```diff
@@ -232,7 +230,10 @@ export class PregelNode<
         triggers: this.triggers,
         mapper: this.mapper,
         writers: [...this.writers, coerceable as ChannelWrite],
-        bound: this.bound.pipe(coerceable),
```

Comment:
This was a regression from a previous PR. It was only possible to do this once `_prepareNextTasks()` was updated.